packages/framework-provider-azure/src/library/api-adapter.ts

```diff
@@ -13,11 +13,15 @@ export interface ContextResponse {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export async function requestSucceeded(body?: any): Promise<ContextResponse> {
+export async function requestSucceeded(
+  body?: any,
```

Comment:
This should be `unknown` not any

Reply:
I only added `headers?: Record<string, number | string | ReadonlyArray<string>>` not changed the body type. Do you think it should be changed?

Reply:
I would try to change it if it is just a 1 liner. If suddenly, this could chain a lot of changes more, so I would keep the PR small in that case and just create tickets / solve it next time.

Reply:
Done